test/test_movies.py

```diff
@@ -1,18 +1,18 @@
-'''
-Example test file
+'''file contains modules to test modules in movies.py'''
 
-New test files should be created for each module
-'''
-# pylint: disable=missing-function-docstring
```

Comment:
If this line is left in, we can ignore the docstrings for the test functions. I think the names and test data should make their purpose clear if setup correctly, I don't think we need docstrings for tests. What do you think?